quartz.config.ts

```diff
@@ -70,6 +70,7 @@ const config: QuartzConfig = {
       Plugin.ContentIndex({
         enableSiteMap: true,
         enableRSS: true,
+        feedDirectories: ["index"], // For a feed for only pages in content/Folder/, add "Folder" to the array
```

Comment:
I think that would be duplicative. Enable should toggle the whole state, with optional finer-grained control through a list.

Reply:
I mean we should have `enableRSS`, and it will do recursive feed for folders, no need to expose this options to users.

For more fine-grained control users can modify the code themselves.

Or at least, we don't show this feedDirectories to `quartz.config.ts`.

Reply:
What's the problem with just generating RSS recursively with `enable RSS`?

My point is that if users want more fine grain per directory, they can do that themselves. By default, we just traverse the vault and generate them, right?

Not sure if there is something that I miss here.

Reply:
enableRSS enables the `index.xml` feed at a minimum. The feature added by this PR is user-side configuration to make additional feeds for specific folders recursively (obviously, if `enableRSS` is disabled, these extra feeds won't be generated because `index.xml` won't either. I don't want users to have to code to make this and I'm not sure why you do. 

[Here's a user that wants the feature](https://discord.com/channels/927628110009098281/1149197588084572161/1159320781633884200). Should they be required to code it or should it just be a config option?

RSS features to improve its usability for the site publisher and the reader shouldn't be gatekept behind programming knowledge.

If you're suggesting that we should just generate a separate feed for *every* directory by default, I'm against that for performance and clutter reasons since default behavior should be what's normal on any RSS-supporting site (a single feed for all pages on the site).

Reply:
Ugh, if they set `enable RSS` to false, yet we still generate RSS per folder, wouldn't that be confusing?

Reply:
I think it should be code. Configuration won't solve everything.

Reply:
also this is my two cent, probably @jackyzha0 would have better input than me here.

Reply:
> Ugh, if they set `enable RSS` to false, yet we still generate RSS per folder, wouldn't that be confusing?

*ohh*, the issue here is a disconnect in what a feed is. some have multiple purposes, but they're only emitted as `.xml` RSS feeds if `enableRSS` is on.

L163: `if (opts?.enableRSS)`
feeds are only written as files if RSS is enabled. Regardless, we still need to generate them because the feed at `feedIndices.get("index")` is used as the sitemap and becomes the `simplifiedIndex` later.

Reply:
Then I suggest that for the index, we make a separate path, since contentIndex will be populated in downstream components.

Reply:
> Then I suggest that for the index, we make a separate path, since contentIndex will be populated in downstream components.

Not sure what you mean by this. what are you referring to with "index"? "index" as a key of feedIndices is going to have the feed for all pages on the site, `index.xml` will be an RSS feed for the content of "index" (only emitted if `enableRSS` is on), and the index contentIndex is used to generate the sitemap and populate simplifiedIndex.

"index" is part of feedIndices because that made the most sense with the structure of this file (and it means I only need one loop to generate all feeds, and it makes configuration by the user easier).